packages/web/components/Search/index.js

```diff
@@ -1,2 +1,51 @@
-export * from './AlgoliaSearchProvider';
-export { default as SearchCard } from './SearchCard';
+import React from 'react';
+import Router from 'next/router';
+import { AlgoliaSearchProvider } from './AlgoliaSearchProvider';
+import { StyledSearchBox, StyledHits, StyledStats } from './styles';
+import SearchCard from './SearchCard';
+
+const Search = () => {
+	let termQuery = '';
```

Comment:
Aqui podemos usar um estado do react, já que vai controlar um comportmento do componente (vai abrir os resultados ou não à medida que for modificado):
```
const [termQuery, setTermQuery] = useState('');
```